src/components/ProfileInfo/ProfileInfo.jsx

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable no-unused-vars */
+import { useForm } from 'react-hook-form';
+import { useState } from 'react';
+import { Icon } from '../UI/Icons/Icons';
+import ProfilePic from '../../assets/images/profile.jpg';
+
+function ProfileInfo() {
+  const [isEdit, setIsEdit] = useState(false);
+  const {
+    register,
+    handleSubmit,
+    resetField,
+    formState: { isValid, errors },
+  } = useForm();
+
+  function handleEditButton() {
+    // eslint-disable-next-line no-unused-expressions
+    isEdit === true ? setIsEdit(false) : setIsEdit(true);
```

Comment:
```suggestion
   setIsEdit(!isEdit);
```
а чего unused-то, вроде вполне used, то есть можно убрать отключение еслинта